packages/web/src/pages/upload-page/components/EditPageNew.tsx

```diff
@@ -72,8 +81,8 @@ export const EditPageNew = (props: EditPageProps) => {
       onSubmit={onSubmit}
       validationSchema={EditTrackSchema}
     >
-      {(formikProps) => (
-        <>
+      {() => (
+        <Form>
```

Comment:
oh dang. whoops!

Reply:
this is sick cause then we dont need an onClick for the button 🔥